docs/samples/index.md

```diff
@@ -14,4 +14,5 @@
 | Deployment  | Spark/Azure Databricks | [Presidio with Spark](deployments/spark/index.md)                                                                                               |
 | Deployment  | Azure Data Factory     | [Presidio in Azure Data Factory with App Service](deployments/data-factory/presidio-data-factory.md#option-1-presidio-as-an-http-rest-endpoint) |
 | Deployment  | Azure Data Factory     | [Presidio in Azure Data Factory with Databricks](deployments/data-factory/presidio-data-factory.md#option-2-presidio-on-azure-databricks)       |
-| Deployment  | Azure Data Factory     | [Presidio in Azure Data Factory Template Gallery](deployments/data-factory/presidio-data-factory-template-gallery.md)                           |
+| Deployment  | Azure Data Factory     | [Presidio as an HTTP service in Azure Data Factory Template Gallery](deployments/data-factory/presidio-data-factory-template-gallery-http.md)                           |
```

Comment:
I think it might be confusing for readers. When would we use the first set of samples (without the gallery) and when should we use the template gallery? If it's already in the template gallery, are the first two samples needed?

Reply:
The ADF samples also include the deployment of ADF, the template gallery deployments have everything but ADF. it may be indeed redundant. wdyt? should I remove them?